.github/actions/deploy-happy-stack/action.yml

```diff
@@ -38,18 +42,24 @@ runs:
         CREATE_TAG: ${{ inputs.create-tag }}
         TAG: ${{ inputs.tag }}
         ENV: ${{ inputs.env }}
+        OPERATION: ${{ inputs.operation }}
       shell: bash
       run: |
         set -ue
         set -o pipefail
         
         IMAGE_TAG="${TAG:-sha-$GITHUB_SHA}"
         echo "Targetting Stack ${STACK_NAME}, tagging with '${IMAGE_TAG}'"
+        HAPPY_DEPLOYMENT_FLAGS=""
 
-        if $(happy list --aws-profile "" --env=${ENV} 2>&1 | grep -q ${STACK_NAME} ); then
-          echo "Updating stack ${STACK_NAME}"
-          happy update --aws-profile "" --env=${ENV} --create-tag=${CREATE_TAG} --tag ${IMAGE_TAG} ${STACK_NAME}
-        else
-          echo "Creating stack ${STACK_NAME}"
-          happy create --aws-profile "" --env=${ENV} --create-tag=${CREATE_TAG} --tag ${IMAGE_TAG} ${STACK_NAME}
+        if [[ ${OPERATION} == "" ]]; then
```

Comment:
```suggestion
        if [[ ${OPERATION} == "create-or-update" ]]; then
```

Reply:
I think it reads better to use the actual string that describes the operation rather than an empty string.